src/navbar.js

```diff
@@ -0,0 +1,33 @@
+const nav = () => {
+  const element = document.createElement('nav');
+  const navClasses = 'navbar navbar-expand-lg navbar-dark bg-dark fixed-bottom navbar-custom-color';
+
+  const navContent = `
+        <div class="d-flex w-50 order-0">
```

Comment:
when the page is loaded (Use document.createElement and document.appendChild
Check my comment....